packages/jsActions/nanoflow-actions-native/src/client/Reload.ts

```diff
@@ -0,0 +1,16 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+/**
+ * @returns {Promise.<void>}
+ */
+export async function reload(): Promise<void> {
+    // BEGIN USER CODE
+    mx.reload();
+    return Promise.resolve();
```

Comment:
Yes that's a good one! 🦉 

Change to a never resolve promise. ✔️

Reply:
I did not understand the need for this at first. I'm glad I found the info here.

It would help future developers understand _why_ a perpetual Promise is needed by adding a comment to the code describing the reasoning.